code/modules/modular_computers/file_system/programs/pai/atmospehere.dm

```diff
@@ -0,0 +1,59 @@
+/datum/computer_file/program/pai_atmos
```

Comment:
might be worth making this a standard program which engineers can download for their mobile computers too, but this can be done in a follow-up PR